### ISSUE TITLE
Feature: Link element in component's data to Elements

### ISSUE THREAD

**Issue by iBobik** [vuejs/vue-devtools#420]
I have a component what will have HTMLElement in it's data:
<img width="583" alt="snimek obrazovky 2017-09-26 v 13 21 02" src="https://user-images.githubusercontent.com/614232/30857959-ffd070ac-a2bd-11e7-9757-f51f4f8f7f68.png">

Will be useful if it will be linked to Elements tab in Devtools, co I can inspect what specific element was pessed there.

**Comment by igidas** [comment]
"Inspect DOM" button you see near component name does that already. 
Doesn't it work for you?

**Comment by iBobik** [comment]
No, because it sends me to this component's element, not to element what is in data variable.

Jan Pobořil
http://honza.poboril.cz/

2. 10. 2017 v 15:06, Kasparas Pochyla <notifications@github.com>:
